src/test/java/com/wikia/webdriver/testcases/desktop/spamwikireview/ViewSpamWikisTests.java

```diff
@@ -0,0 +1,102 @@
+package com.wikia.webdriver.testcases.desktop.spamwikireview;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.communities.desktop.pages.SpamWikiReviewPage;
+
+import org.apache.commons.lang3.EnumUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+@Test(groups = "SpamWikiReview")
+public class ViewSpamWikisTests extends NewTestTemplate {
+
+  /**
+   * Tests if a normal user correctly doesn't have access to Spam Wiki Review
+   */
+  @Test
+  @Execute(asUser = User.USER)
+  public void authorizationFailSpamWikiReviewTest() {
+    SpamWikiReviewPage spamWikiReviewPage = new SpamWikiReviewPage();
+    spamWikiReviewPage.open();
+
+    Assertion.assertTrue(spamWikiReviewPage.getRawVisibleBodyText()
+                             .matches(".*(User not authorized|Unauthorized).*"),
+                         "No 'unauthorized' type message was displayed to the user");
+
+    Assertion.assertTrue(spamWikiReviewPage.getListDisplayedWikisTableRows().isEmpty(),
+                         "It shouldn't be possible to see Wikis' list when unauthorized");
+  }
+
+  @Test
+  @Execute(asUser = User.STAFF)
+  public void filterJapaneseLanguage() {
+    SpamWikiReviewPage spamWikiReviewPage = new SpamWikiReviewPage();
+    spamWikiReviewPage.open();
+
+    String spamWikiReviewListViewUrl = spamWikiReviewPage.getCurrentUrl();
+
+    // select 'ja' language
+    spamWikiReviewPage.selectLanguageOfWikis(SpamWikiReviewPage.LANGUAGE_CODE.ja);
+
+
+    // assert current page contains query param selecting the language
+    Assertion.assertEquals(spamWikiReviewPage.getCurrentUrl(),
+                           spamWikiReviewListViewUrl + "?lang=" + SpamWikiReviewPage.LANGUAGE_CODE.ja);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "?lang=" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)